solana/indexer/common/checkpoints.go

```diff
@@ -1,18 +1,69 @@
-package indexer
+package common
 
 import (
 	"context"
 	"crypto/sha256"
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
+	"time"
 
 	"api.audius.co/database"
 	"github.com/jackc/pgx/v5"
 	pb "github.com/rpcpool/yellowstone-grpc/examples/golang/proto"
+	"go.uber.org/zap"
 )
 
-func insertBackfillCheckpoint(ctx context.Context, db database.DBTX, fromSlot uint64, toSlot uint64, address string) (string, error) {
+const MAX_SLOT_GAP = 2500
```

Comment:
nit: Comment for why this number (something something 3000 slot limit on river of lava, etc)